.github/workflows/pr-auto-assign.yml

```diff
@@ -1,13 +1,26 @@
-name: 'Auto Assign'
+name: 'Auto assign PR reviewers'
 on:
   pull_request_target:
     types: [opened, ready_for_review, reopened]
 
+permissions:
+  pull-requests: write
+
 jobs:
   add-reviews:
     if: ${{ join(github.event.pull_request.requested_reviewers.*.login, ',') == '' }}
     runs-on: ubuntu-latest
     steps:
-      - uses: kentaro-m/auto-assign-action@6d966a1f3bd73adfe44dd19dc42cc5845e39ebd3 # v1.2.4
+      - uses: actions/github-script@v6
         with:
-          configuration-path: ".github/pr-auto-assign-config.yml"
+          script: |
+            const reviewers = ['mstoykov', 'imiric, 'codebien', 'olegbespalov', 'oleiade'];
+            const reviewerCount = 2;
+
+            const reviewersWithoutAuthor = reviewers.filter(user => user !== context.payload.pull_request.user.login);
+            github.rest.pulls.requestReviewers({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: context.payload.pull_request.number,
+              reviewers: assigneesWithoutAuthor.sort(() => 0.5 - Math.random()).slice(0, reviewerCount),
```

Comment:
Do you know if this will only do the assignment if the PR is marked as "ready for review", i.e. not for draft PRs? If not, we should ignore the assignment for draft PRs.

Reply:
Hmm I am not sure, https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request doesn't seem to document this explicitly. I suggest we try it like this and adjust if needed. 

My guess is that just the `ready_for_review` event won't trigger if I open a new *non-draft* PR, so we need all 3. The actual solution might be to add some `if` JavaScript check in the job body to short-circuit it and return early if the PR is in a draft state :thinking: